go/mysql/binlog_event_mariadb.go

```diff
@@ -87,8 +87,8 @@ func (ev mariadbBinlogEvent) StripChecksum(f BinlogFormat) (BinlogEvent, []byte,
 		// Checksum is the last 4 bytes of the event buffer.
 		data := ev.Bytes()
 		length := len(data)
-		checksum := data[length-4:]
-		data = data[:length-4]
+		checksum := data[length-binlogChecksumLen:]
+		data = data[:length-binlogChecksumLen]
```

Comment:
again, we must not assume `CRC32` here.

Reply:
Again, sure. But it's not related to this PR, I only made the 4 less "magical". 🙂

Reply:
It's also worth noting that the checksum is either 0 bytes, when there is no checksum, or it's 4 bytes regardless of the algorithm used. In both MySQL and MariaDB.

Reply:
:+1: